examples/src/app/page.tsx

```diff
@@ -7,7 +7,7 @@ type HomeProps = {
 
 export default async function Home({ searchParams }: HomeProps) {
   return (
-    <div className="space-y-4 p-4 lg:p-8">
+    <div className="space-y-4 p-4 lg:p-8 relative w-auto bg-[radial-gradient(at_bottom_left,_var(--tw-gradient-stops))] from-white/90 via-pink-50/90 to-pink-100/90 rounded-lg">
```

Comment:
I think we need to add `h-full` here to make sure this container fills the height of the page.